css/styles.css

```diff
@@ -1,43 +1,91 @@
-.yellow {
-  background-color: yellow;
-}
+@media only screen and (min-width: 768px) {
+  .yellow {
+    background-color: cyan;
+  }
 
-.container {
-  margin-top: 50px;
-  margin-bottom: 50px;
-}
+  .container {
+    margin-top: 50px;
+    margin-bottom: 50px;
+  }
 
-.header {
-  background-color: blue;
-  height: 250px;
-}
+  .header {
+    background-color: red;
+    height: 250px;
+  }
 
-.nav {
-  background-color: green;
-  height: 100px
-}
+  .nav {
+    background-color: blue;
+    height: 100px
+  }
 
-.content {
-  background-color: red;
-  height: 300px;
-}
+  .content {
+    background-color: orange;
+    height: 300px;
+  }
 
-.aside {
-  background-color: cyan;
-  height: 300px;
-}
+  .aside {
+    background-color: purple;
+    height: 300px;
+  }
 
-.promo {
-  background-color: orange;
-  height: 150px;
-}
+  .promo {
+    background-color: chartreuse;
+    height: 150px;
+  }
+
+  .footer {
+    color: white;
+    background-color: yellow;
+    height: 50px;
+  }
 
-.footer {
-  color: white;
-  background-color: black;
-  height: 50px;
+  .block {
+    border: 3px solid black;
+  }
 }
 
-.block {
-  border: 3px solid black;
+@media only screen and (min-width: 875px) {
+  .yellow {
+    background-color: yellow;
+  }
```

Comment:
Great stuff! Quick suggestion (for best practices): it is best to place the media queries at the bottom of your file only because CSS is read from top to bottom. We want it to care at the very end when it has read the _original_ scripts first. other than that, GREAT JOB!